psalm-baseline.xml

```diff
@@ -1,60 +1,120 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<files psalm-version="dev-master@">
+<files psalm-version="5.26.1@d747f6500b38ac4f7dfc5edbcae6e4b637d7add0">
+  <file src="src/AuthenticationService.php">
+    <PossiblyNullArgument>
+      <code><![CDATA[$provider]]></code>
+    </PossiblyNullArgument>
+    <RiskyTruthyFalsyComparison>
```

Comment:
I would simply disable this `RiskyTruthyFalsyComparison` in the main config file for all repos, it's pretty annoying.